src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/create-cluster/create-cluster-review.component.html

```diff
@@ -10,10 +10,21 @@
           <td>{{ hostsCount }}</td>
         </tr>
         <tr>
-          <td i18n
-              class="bold">Storage Capacity</td>
-          <td><span i18n>Number of devices: {{ totalDevices }}. Raw capacity:
-            {{ totalCapacity | dimlessBinary }}.</span></td>
+          <td>
+          <dl>
+            <dt>
+              <p i18n>Storage Capacity</p>
+            </dt>
+            <dd>
+              <p i18n>Number of devices</p>
+            </dd>
+            <dd>
+              <p i18n>Raw capacity</p>
+            </dd>
+          </dl>
+        </td>
+          <td class="indent"><p>{{ totalDevices }}</p><p>
```

Comment:
Applying this change will give you "almost" the same result. And probably you can remove the css change.
```suggestion
          <td class="pt-5"><p>{{ totalDevices }}</p><p>
```

Reply:
Done.! Thanks @nizamial09